docs-chef-io/content/inspec/cli.md

```diff
@@ -188,6 +188,12 @@ inspec compliance login
 inspec exec compliance://username/linux-baseline
 ```
 
+`inspec automate` is an alias for `inspec compliance` and works the same way:
```

Comment:
List automate before compliance, and present compliance as being a backwards-compatible alias.